packages/tools/src/types/EventTypes.ts

```diff
@@ -49,10 +49,6 @@ type AnnotationModifiedEventDetail = {
  * The data that is passed to the event handler when an annotation is completed drawing.
  */
 type AnnotationRemovedEventDetail = {
-  /** unique id of the viewport */
-  viewportId: string;
-  /** unique id of the rendering engine */
-  renderingEngineId: string;
   /** The annotation that is being added to the annotations manager. */
```

Comment:
Thinking about this, I think we need the `FrameOfReferenceSpecificAnnotationManager`'s UID? For most cases this will be the default UID, but in the case of still supporting element specific state management it is required to uniquely identify the annotation?

It still keeps it decoupled from the view, as the only information in the event is "This is the annotation, and this is the annotation manager it belongs to."